jib-core/src/test/java/com/google/cloud/tools/jib/image/json/JsonToImageTranslatorTest.java

```diff
@@ -85,7 +85,7 @@ public void testToImage_oci()
     testToImage_buildable("core/json/ocimanifest.json", OciManifestTemplate.class);
   }
 
-  @Test
+  @Test(expected = Test.None.class /* no exception expected */)
```

Comment:
This is an interesting discovery and approach! But maybe let's stick with directly calling `assert` statements  to follow the guidelines on having dedicated `arrange`, `act` and `assert` blocks in tests?

Reply:
make sense. Let me do the changes and stick to `assert` statements for these.